riemann_client/transport.py

```diff
@@ -176,20 +176,27 @@ class BlankTransport(Transport):
     ``riemann_client/tests/test_riemann_command.py``.
     """
 
+    def __init__(self, *args, **kwargs):
+        self.messages = []
+
     def connect(self):
         """Creates a list to hold messages"""
-        self.messages = []
+        pass
 
     def send(self, message):
         """Adds a message to the list, returning a fake 'ok' response
 
         :returns: A response message with ``ok = True``
         """
-        self.messages.append(message)
+        for event in message.events:
+            self.messages.append(event)
```

Comment:
I'm fine with it storing the events instead - I don't think there's anything else Riemann would keep from a message - but the code should probably change to use `self.events` in that case so it's clear what it's doing.